README.md

```diff
@@ -32,7 +32,7 @@ Choices of Capstone App - Part. 2
 
 Supporting Materials
 --------------------
-* [Student Cheatsheet](outline/cheatsheet.md) | [PDF](ClojureBridgeCheatsheet-v1.pdf)
+* [Student Cheatsheet](outline/cheatsheet.md) | [PDF](ClojurebridgeCheatsheet-v2.pdf)
```

Comment:
We probably shouldn't have version numbers in our file names, but that's a separate issue. ;)